modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -1980,11 +1990,38 @@ struct Mapper {
         return u;
     }
 };
+}  // namespace linear
+
+namespace linear32f {
+struct Mapper {
+    typedef float alpha_type;
+    typedef int   index_type;
+    constexpr static const float unity = 1;
+
+    typedef MapperUnit<float, int> Unit;
+
+    static inline Unit map(double ratio, int start, int max, int outCoord) {
+        float f = static_cast<float>((outCoord + 0.5) * ratio - 0.5);
```

Comment:
i may be wrong, please correct me, but I suppose that `0.5` relates relates to `double` type
and `outCoord + 0.5` would upgraded to `double`  then with `static_cast<float>` we reduced it to `float`. I suppose such conversation is not free for us (critical performance path?)
What do you think to use explicit `0.5f` without static_cast ?

Reply:
@sivanov-work The `ratio` function argument is used as multiplier in this formula and also it has `double` type, so all expression has double type. Therefore static_cast<float> is necessary anyway(no matter if 0.5 or 0.5f I use).

Reply:
ok, missed that ratio is double